client/verta/docs/examples/tutorials.rst

```diff
@@ -33,6 +33,7 @@ Core Verta functionality
 
 * `Using the Verta Web App <tutorials/webapp.html>`_
 * `Workspaces in Verta <tutorials/workspaces.html>`_
+* `Using a Client Config File <tutorials/config.html>`__
```

Comment:
```suggestion
* `Using a Client Config File <tutorials/config.html>`_
```

Reply:
I used two underscores on purpose, but I'll just fix all of them later in another PR.